### DISCUSSION TITLE
Well, this sounds like a great time to deprecate my own options for this!

### DISCUSSION THREAD

**Discussion by jpmurray** [spatie/laravel-disable-floc#2] [Show and tell]
I mean, there's not way I can [maintain one](https://github.com/jpmurray/LaravelNoFLoC) when Spatie does it too ! 🤷‍♂️

**Comment by freekmurze** [comment]
O, I didn't see yours when searching packagist for a Laravel FLoC package.

O well, happy to take the maintainence burden of your hands. 🙂

**Reply by jpmurray** [reply]
I didn't see any trouble in the future, but yeah, I'll gladly pitch maintenance on your hands ;-) I'll be deprecating it in the coming days and recommending this one 👍
